README.md

```diff
@@ -418,6 +418,56 @@ npm install prosemirror-utils
    ```
 
 
+ * **`forEachCellInColumn`**`(columnIndex: number, cellTransform: fn(cell: {pos: number, node: ProseMirrorNode}) → fn(tr: Transaction) → Transaction) → fn(tr: Transaction) → Transaction`\
+   Returns a new transaction that maps a given `cellTransform` function to each cell in a column at a given `columnIndex`.
+   It will set the selection into the last cell in the column.
```

Comment:
Probably want to make cell selection optional?

Reply:
yeah I thought about it too, I'll add a 3rd param then.

Reply:
done